qa.cfg

```diff
@@ -1,10 +1,14 @@
 [buildout]
 index = https://pypi.org/simple/
 package-min-coverage = 80
+versions = versions
 parts +=
     code-analysis
     i18ndude
 
+[versions]
+configparser = <5.0.0  # configparser only needed for Py<3.
```

Comment:
@thet take a look here:

https://github.com/collective/buildout.plonetest/blob/09a6fa6ad9387f8932a92578e337ee0643c395c9/plone-5.2.x.cfg#L25

Reply:
@thet Can you try this suggestion? So only pin `configparser` on python27 in `qa.cfg`.